diff --git a/lua/plugins/lsp.lua b/lua/plugins/lsp.lua
index bd64663..25605ea 100644
--- a/lua/plugins/lsp.lua
+++ b/lua/plugins/lsp.lua
@@ -120,7 +120,7 @@ autocmd('LspAttach', {
     desc = 'Disable semantic highlight',
 })
 
-local enabled_lsps = { 'r', 'python', 'bash', 'cpp', 'vim', 'nvim', 'pinyin', 'sql', 'latex', 'go', 'rust', 'efm' }
+local enabled_lsps = { 'python', 'nvim', 'efm' }
 
 local lsp_configs = {}
 
