diff --git a/lua/plugins/cli_tools.lua b/lua/plugins/cli_tools.lua
index e268256..44c3592 100644
--- a/lua/plugins/cli_tools.lua
+++ b/lua/plugins/cli_tools.lua
@@ -36,8 +36,13 @@ return {
             local yarepl = require 'yarepl'
             local aider = require 'yarepl.extensions.aider'

+            aider.setup {
+                aider_cmd = { 'fhs', 'aider' },
+            }
+
             yarepl.setup {
                 metas = {
+                    ipython = { cmd = { 'fhs', 'ipython' } },
                     aider = aider.create_aider_meta(),
                     python = false,
                     R = false,
