diff --git a/lua/plugins/cli_tools.lua b/lua/plugins/cli_tools.lua
index b834d6b..db18ecb 100644
--- a/lua/plugins/cli_tools.lua
+++ b/lua/plugins/cli_tools.lua
@@ -69,8 +69,17 @@ return {
             local yarepl = require 'yarepl'
             local aider = require 'yarepl.extensions.aider'
 
+            aider.setup {
+                aider_cmd = { 'fhs', '-c', "'aider'" },
+            }
+
             yarepl.setup {
-                metas = { aider = aider.create_aider_meta(), python = false, R = false },
+                metas = {
+                    aider = aider.create_aider_meta(),
+                    python = false,
+                    R = false,
+                    ipython = { cmd = { 'fhs', '-c', "'ipython'" } },
+                },
             }
         end,
         init = function()
